daml-assistant/integration-tests/src/Main.hs

```diff
@@ -59,15 +59,10 @@ quickstartTests :: FilePath -> FilePath -> TestTree
 quickstartTests quickstartDir mvnDir = testGroup "quickstart"
     [ testCase "daml new" $
           callProcessQuiet "daml" ["new", quickstartDir]
-    , testCase "daml init" $ withCurrentDirectory quickstartDir $
-          callProcessQuiet "daml" ["init"]
-    , testCase "daml package" $ withCurrentDirectory quickstartDir $
-          -- This location is assumed by the codegen in the quickstart example.
-          callProcessQuiet "daml" ["package", "-o", "target/daml/iou.dar"]
```

Comment:
Removing this breaks the `mvn compile` test below which assumes this location. You could just add the `-o` parameter to `daml build` instead.

Reply:
Ah, thanks!